### PR TITLE
[1.21.1] Update Icons

### PR DESCRIPTION
**Pull request by UnRealDinnerbone**

### CODE REVIEW

File: common/src/main/resources/assets/ftblibrary/textures/icons/bell.png
Comment (MichaelHillcox) [inline common/src/main/resources/assets/ftblibrary/textures/icons/bell.png:1]: This file should be renamed?

File: common/src/main/resources/assets/ftblibrary/textures/icons/blue_button.png
Comment (MichaelHillcox) [inline common/src/main/resources/assets/ftblibrary/textures/icons/blue_button.png:1]: What's the difference between this and `blank_blue`? Also, is this needed anymore?

File: common/src/main/resources/assets/ftblibrary/textures/icons/cancel.png
Comment (MichaelHillcox) [inline common/src/main/resources/assets/ftblibrary/textures/icons/cancel.png:1]: `cancel` and `close` should like be merged?

File: common/src/main/resources/assets/ftblibrary/textures/icons/diamond.png
Comment (MichaelHillcox) [inline common/src/main/resources/assets/ftblibrary/textures/icons/diamond.png:1]: Can the game one not work?

File: common/src/main/resources/assets/ftblibrary/textures/icons/dice.png
Comment (MichaelHillcox) [inline common/src/main/resources/assets/ftblibrary/textures/icons/dice.png:1]: I like this one but I think the dots need to be clear on the new version

File: common/src/main/resources/assets/ftblibrary/textures/gui/missing_image.png
Comment (MichaelHillcox) [inline common/src/main/resources/assets/ftblibrary/textures/gui/missing_image.png:1]: Why is this 128x128?

**Review by MichaelHillcox — APPROVED** [review]
Just a few comments

**Comment by Jake-E** [comment]
@UnRealDinnerbone How are we looking here?

**Comment by desht** [comment]
I spotted one little problem with the lock.png and lock_open.png - they're a pixel lower than other icons (i.e. most icons have a 1-pixel gap at the bottom - the lock icons don't). Not a big thing, but it looks bad in this FTB Chunks screen:

image

May be tricky to fix, since lock_open occupies the full 16 vertical pixels of the icon. A slightly janky workaround might be just to place the icon 1 pixel higher in code?

Update: for now, doing the code change in FTB Chunks. Looks fine.

**Pull request merged**